### DISCUSSION TITLE
GitHub student developer pack

### DISCUSSION THREAD

**Discussion by tanzeem131** [community/community#137710] [GitHub Education]
### Hi everyone,

Access free GitHub Education benefits is approved but i am unable to login to frontend master course because the status on education.github.com is still showing "Congrats! Your request for [@tanzeem131] was approved and your benefits will be available [soon]." after 3 days.
Kindly look  into this matter.

**Comment by davevad93** [comment]
Hi @tanzeem131 ,

You have to wait for your benefits to be available before you can redeem the pack offers, when they are available the status label will turn purple and change to:

- `Congrats! Your request for @tanzeem131 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
